hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/utils/TestFederationStateStoreFacade.java

```diff
@@ -312,10 +312,10 @@ public void testUpdateNewToken() throws YarnException, IOException {
     facade.updateStoredToken(dtId1, renewDate1);
     token1.put(dtId1, renewDate1);
 
-    MemoryFederationStateStore stateStore =
+    MemoryFederationStateStore federationStateStore =
```

Comment:
Why changing the name? It is pretty clear in this context.

Reply:
Thank you very much for your help reviewing the code.

checkStyle reported a compilation problem

```
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-
common/src/test/java/org/apache/hadoop/yarn/server/federation/utils/TestFederationStateStoreFacade.java:315:    
MemoryFederationStateStore stateStore =:32: 'stateStore' hides a field. [HiddenField]

./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-
common/src/test/java/org/apache/hadoop/yarn/server/federation/utils/TestFederationStateStoreFacade.java:341:    
MemoryFederationStateStore stateStore =:32: 'stateStore' hides a field. [HiddenField]
```